CONTRIBUTING.md

```diff
@@ -43,6 +43,15 @@ a "soft" commitment.
 If you're eager to increase the velocity of the dev process, reviewing other contributors work is
 the best you can do while waiting review on yours.
 
+Getting Project Context
```

Comment:
I'm not sure if "context" best encapsulates all these ideas. They seem more about understanding developer or contributor interaction.

The word "context" is also used in the "Getting Start" section. Did you mean for these to mean the same thing? I think of context more in terms of understanding how the project is intended to be used.

Reply:
I see, I do have a different understanding of context  as : "Contributors practices and development process shaped by project history". But given your confusion, I think a better term is "Culture". Re-worded.

Your definition of "context" means more "Design Goals" to me :)

Reply:
To a broader point, does this need to be separate section? There seems to be a lot of overlap between this and the "Getting Started" section.

Reply:
Yeah makes sense